package.json

```diff
@@ -52,7 +50,9 @@
     "typescript": "^3.8.2"
   },
   "dependencies": {
+    "@types/swagger-schema-official": "^2.0.20",
```

Comment:
this should still remain in devDep since it's just for ts

Reply:
Oops, you're right. Will correct.